### PR TITLE
Better Server Control Area (References #516)

### LINKED ISSUE #516: Server power controls
It might just be me, but I have a few minor suggestions for this area
image

- I'm not sure I would show the "stop", "kill", "pause", and "restart" buttons when the server is offline.
- If the stop button is red, I'd make the start button green.

### PR DESCRIPTION
**Pull request by Dylan3304**
Please see #516. Added everything that the OP requested.

### CODE REVIEW

File: app/views/node/index.html [hunk 1]
```diff
@@ -95,10 +95,10 @@ <h3 class="nopad">{{ l.render('node.overview.players_h5') }}</h3><hr />
...
 					<hr />
 					{% if permission.has('console.power') %}
-						<button class="btn btn-primary btn-sm start" id="on">{{ l.render('string.start') }}</button>
-						<button class="btn btn-primary btn-sm restart" id="restart">{{ l.render('string.restart') }}</button>
-						<button class="btn btn-danger btn-sm stop" id="off">{{ l.render('string.stop') }}</button>
-						<button class="btn btn-primary btn-sm" data-toggle="modal" data-target="#pauseConsole" id="pause_console"><small><i class="fa fa-pause"></i></small></button>
+						<button class="btn btn-success btn-sm start" id="on">{{ l.render('string.start') }}</button>
+						<button class="btn btn-primary btn-sm restart" disabled="disabled" id="restart">{{ l.render('string.restart') }}</button>
+						<button class="btn btn-danger btn-sm stop" disabled="disabled" id="off">{{ l.render('string.stop') }}</button>
+						<button class="btn btn-primary btn-sm" disabled="disabled" id="pause_console_btn" data-toggle="modal" data-target="#pauseConsole" id="pause_console"><small><i class="fa fa-pause"></i></small></button>
```
Comment (LordRalex) [inline app/views/node/index.html]: Why does this have 2 ids now

File: app/views/node/index.html [hunk 2]
```diff
@@ -128,6 +128,11 @@ <h4 class="modal-title" id="PauseConsole">{{ l.render('node.console.scrollstop_h
 	</div>
 </div>
 <script type="text/javascript">
+	socket.on('query', function (data) {
+		document.getElementById("off").removeAttribute("disabled");
```
Comment (LordRalex) [inline app/views/node/index.html]: Why did you use the document over jquery?

**Comment by Wargog** [comment]
The disabled attribute doesn't have to have a value, in HTML5 it needs to be used it without a value.

**Comment by LordRalex** [comment]
Also, please use related commit messages. Simply saying "resolve X" does not tell us what it is doing directly. Please include what the change has done for the commit message, and have Resolve X be a part of that message.

**Comment by DaneEveritt** [comment]
Couple of issues with this:

1.) The socket query is sent every 10 seconds, so you're going to have quite the delay in showing the buttons.
2.) What happens if the server is stopped? Your code simply disables elements, doesn't handle anything else.
3.) This PR should be a single squashed commit, not multiple commits.

**Comment by Dylan3304** [comment]
I'm going to try this again in a new pull. I'm trying to help, and got to carried away.

**Comment by DaneEveritt** [comment]
Go ahead and keep it in this pull request, you can squash the commits and it will show up correctly here.

**Pull request closed without merging**